website/src/theme/DocItem/index.js

```diff
@@ -0,0 +1,203 @@
+/**
```

Comment:
Does Docosaurus support using the `.jsx` extension for JSX files?

Reply:
Docusaurus always uses `.js` extensions, don't know why. If we want to use `.jsx` we'd have to update the other files too so I'd suggest we do it on a separate PR.

Reply:
We should do this though since it 

1) plays nicer with editors
2) these files are technically not JavaScript, they are JSX


FB plays fast and loose with the truth when it comes to content type. They also called `Flow` files `.js` which was another case of something that is not true 🤷🏻‍♂️